src/app/cube/home/learning-object-info/learning-outcomes/learning-outcomes.component.scss

```
@@ -1,15 +1,78 @@
-.content{
+@import 'vars.scss';
+
+.content {
     margin-top: 50px;
     margin-right: 50px;
+    padding-left: 0px;
     display: flex;
     flex-direction: column;
-    .taxonomy{
-      margin-bottom: 400px;
+    
+    .blooms-cards {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        justify-content: space-evenly;
+        gap: 25px;
+        color: $homepage-light-blue;
+        padding-bottom: 50px 0 300px 0;
+
+        .blooms-card {
+            padding: 20px 40px;
+            min-width: 175px;
+            max-width: 175px;
+            height: 100px;
+            background-color: $primary-white;
+            border: 1px solid $secondary-white;
+            border-radius: 8px;
+            transform-style: preserve-3d;
+            position: relative;
+
+            .blooms-face {
+                display: block;
+                position: absolute;
+                text-align: center;
+                width: 175px;
+                backface-visibility: hidden;
+            }
+
+            .blooms-content {
+                display: flex;
+                flex-direction: column;
+                align-items: center;
+                max-height: 100px;
+                cursor: default;
+                -webkit-transform : rotateY(0deg);
+                -ms-transform     : rotateY(0deg);
+                transform         : rotateY(0deg);
```

```suggestion
                transform         : rotateY(0deg);
                justify-content: center;
```

Continuation of card change suggestion